src/annotate.ts

```diff
@@ -61,10 +66,9 @@ export default async function annotate(opts: { payload: { field: string, term: s
       ontologyShortName // shortname from IRI
 
     };
-
+    //console.log(result);
```

Comment:
I would avoid logging to the std output if possible during runtime.

Reply:
Yes. it is commented out.

Reply:
Sorry 😅